apps/design-system/src/components/OperationIcon.stories.tsx

```diff
@@ -1,14 +1,15 @@
 import { OperationIcon } from '@asyncapi/studio-ui';
 
-export default {
-    component: OperationIcon,
-    parameters: {
-        backgrounds: {
-          default: 'dark'
-        }
-    }
+const meta = {
```

Comment:
What's the advantage of this change? 🤔 I honestly prefer the straight `export default` than having to define a variable and then export it.

Reply:
eslint recommends naming each export.
https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/no-anonymous-default-export.md

I see we are not currently using that name in the  code but we might reuse some parts of the meta object in it's stories in other components. it would just add consistency between stories. but still, not a big deal 😆 I can disable this rule.

Reply:
🤷‍♂️😄